src/explore-education-statistics-frontend/src/modules/methodologies/MethodologyIndexPage.tsx

```diff
@@ -28,14 +28,14 @@ class MethodologyIndexPage extends Component<Props> {
     const { themes } = this.props;
     return (
       <Page
-        breadcrumbs={[
-          {
-            link: '/methodology',
-            name: 'Methodology',
-          },
-        ]}
+        title="Education statistics: methodology"
```

Comment:
Change this to Methodologies (the original title was put in as a placeholder by James and should not be trusted as true content). The breadcrumbs/meta should also respect this title.